recipes/pyrolite/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pyrolite" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyrolite-{{ version }}.tar.gz
+  sha256: f43a2a715db325340b45ddbfbae49aab15d00c998d0482a59a8030f2616e3276
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - joblib
+    - matplotlib-base >=3.5
+    - mpltern >=0.3.4
+    - mpmath
+    - numpy
+    - numpydoc
+    - openpyxl
+    - pandas >=1.0
+    - periodictable
+    - psutil
+    - requests
+    - scipy >=1.2
+    - sympy >=1.7
+    - tinydb >4.1
+    - typing-extensions
+    - xlrd
+
+test:
+  imports:
+    - pyrolite
+    - pyrolite.plot
+
+about:
+  home: https://github.com/morganjwilliams/pyrolite
+  summary: Tools for geochemical data analysis.
+  license: MIT AND BSD-3-Clause
```

Probably better to do:
```suggestion
  license: LicenseRef-CSIRO-Open-Source
```
since there's so much legalese at the bottom.